app/views/tickets/edit.html.slim

```diff
@@ -0,0 +1,20 @@
+.row
+  .card.w-100
+    .card-body
+      h5.card-title Edit ticket
+      / = @ticket.place
+      = form_for @ticket, method: :patch do |f|
+        .form-group
+          label for="place" Place
+          = f.text_field :place, class: 'form-control', id: 'place', placeholder: "Enter place yours ticket"
```

Comment:
nie jestem pewien co znaczy `Enter place yours ticket`, czy to miało chodzić o miejsce wydarzenia, na które jest bilet, czy może o miejsce na wydarzeniu? Może spróbuj jakoś ładniej to napisać

Reply:
Zmieniłem na "Enter sector"